src/Tizen.NUI/src/public/BaseComponents/DirectRenderingGLView.cs

```diff
@@ -0,0 +1,258 @@
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// DirectRenderingGLView allows drawing with OpenGL. You can render to a Window directly.
+    /// DirectRenderingGLView creates a context.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DirectRenderingGLView : View
+    {
+        /// <summary>
+        /// The parameter of the RenderFrame Callback.
+        /// It has data to render directly.
+        /// </summary>
+        [SuppressMessage("Microsoft.Design", "CA1034:NestedTypesShouldNotBeVisible")]
```

Comment:
Is there any reason using public nested class?
It is not recommanded. 
If it is for grouping, applying new namespace is more commonly used.

Reply:
Thank you for your opinion.
This is backport PR. I'm not sure that I should fix it here.
This class is only used in the GLRenderFrameDelegate. 
And DirectRenderingGLView.OnRenderFrame creates the object, the data is passed from Dali.

I'm not familiar with the situation and nested class. If I have to modify it, I'll check it.